diff -Naur --strip src/animation.ts src/animation.ts
--- src/animation.ts	2021-08-19 01:16:20.000000000 +0200
+++ src/animation.ts	2021-08-19 01:17:39.000000000 +0200
@@ -145,23 +145,23 @@
 	skin.rightLeg.rotation.x = Math.sin(time + Math.PI) * 0.5;
 
 	// Arm swing
-	skin.leftArm.rotation.x = Math.sin(time + Math.PI) * 0.5;
-	skin.rightArm.rotation.x = Math.sin(time) * 0.5;
-	const basicArmRotationZ = Math.PI * 0.02;
-	skin.leftArm.rotation.z = Math.cos(time) * 0.03 + basicArmRotationZ;
-	skin.rightArm.rotation.z = Math.cos(time + Math.PI) * 0.03 - basicArmRotationZ;
+	skin.leftArm.rotation.x = Math.sin(time + Math.PI) * 0.3;
+	skin.rightArm.rotation.x = Math.sin(time) * 0.3;
+	//const basicArmRotationZ = Math.PI * 0.02;
+	//skin.leftArm.rotation.z = Math.cos(time) * 0.03 + basicArmRotationZ;
+	//skin.rightArm.rotation.z = Math.cos(time + Math.PI) * 0.03 - basicArmRotationZ;
 
 	// Head shaking with different frequency & amplitude
-	skin.head.rotation.y = Math.sin(time / 4) * 0.2;
-	skin.head.rotation.x = Math.sin(time / 5) * 0.1;
+	//skin.head.rotation.y = Math.sin(time / 4) * 0.2;
+	//skin.head.rotation.x = Math.sin(time / 5) * 0.1;
 
 	// Ears should mirror head
 	ears.rotation.y = skin.head.rotation.y
 	ears.rotation.x = skin.head.rotation.x
 
 	// Always add an angle for cape around the x axis
-	const basicCapeRotationX = Math.PI * 0.06;
-	player.cape.rotation.x = Math.sin(time / 1.5) * 0.06 + basicCapeRotationX;
+	const basicCapeRotationX = Math.PI * 0.09;
+	player.cape.rotation.x = Math.sin(time / 1.5) * 0.09 + basicCapeRotationX;
 };
 
 export const RunningAnimation: Animation = (player, time) => {
